src/internal/hooks/use-unique-id/index.ts

```diff
@@ -14,5 +14,5 @@ const useIdFallback = () => {
 const useId: typeof useIdFallback = (React as any).useId ?? useIdFallback;
 
 export function useUniqueId(prefix?: string) {
-  return `${prefix ? prefix : ''}` + useId();
+  return `${prefix ? prefix : 'i'}` + useId();
```

Comment:
I recommend adding a comment as to why the unique id must start with a letter. This might help future readers avoid confusion (Why is the unique id prefixed with an `i`?).